src/http_rest_api_handler.cpp

```diff
@@ -267,7 +266,7 @@ Status HttpRestApiHandler::processPipelineRequest(const std::string& modelName,
 
     tensorflow::serving::PredictRequest& requestProto = requestParser.getProto();
     requestProto.mutable_model_spec()->set_name(modelName);
-    status = getPipeline(ModelManager::getInstance(), pipelinePtr, &requestProto, &responseProto);
+    status = ModelManager::getInstance().getPipeline(pipelinePtr, &requestProto, &responseProto);
```

Comment:
I am pretty sure you will block testability with such change.

Reply:
I also thought so, but apparently this change requires only simple adjustment of function calls in unit tests and it seems to be working.